docs/app/documentation/component-docs/alert/examples/alert-width-example.component.html

```diff
@@ -0,0 +1,15 @@
+<fd-alert [type]="'warning'" [width]="'250px'">
+    A dismissible warning-type alert with customizable width of 250px.
+</fd-alert>
+
+<fd-alert [type]="'success'" [width]="'50%'" [dismissible]="false">
+    A non-dismissible success-type alert with customizable width of 50%.
+</fd-alert>
+
+<fd-alert [type]="'information'" [width]="'550px'">
+    A dismissible information-type alert with customizable width of 550px.
+</fd-alert>
+
+<fd-alert [type]="'error'" [width]="'100%'" [dismissible]="false">
```

Comment:
You don't need to specify [width]="'100%'" for this example. It's the default value.